src/app/lib/access/dad_jokes/mocks/dad_jokes_list_mock.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:app/access/dad_jokes/data/dad_joke_content_data.dart';
+import 'package:app/access/dad_jokes/mocks/dad_jokes_data_mock.dart';
+
+List<DadJokeContentData> getDadJokesList() {
```

Comment:
Shouldn't this be `getMockedDadJokesList` to be consistent with the other method?